
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Tools &#8212; Brendan  documentation</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Bio with Python" href="PythonBIO.html" />
    <link rel="prev" title="Graph" href="Graph.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Brendan  documentation</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search the docs ..." aria-label="Search the docs ..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Contents:
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="Work.html">
   My work
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Graph.html">
   Graph
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Tools
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="PythonBIO.html">
   Bio with Python
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Data.html">
   Data
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="RCC.html">
   RCC
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Template.html">
   Template
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="_sources/Tools.rst.txt"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.rst</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#assembler">
   Assembler
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#flye">
     Flye
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#miniasm">
     miniasm
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quickmerge">
   quickmerge
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#trimmer">
   Trimmer
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#nanofilt">
     Nanofilt
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#porechop">
     Porechop
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#graph">
   Graph
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#vg">
     VG
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#minigraph">
     minigraph
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#pggb">
     pggb
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#alignment">
   Alignment
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#minimap2">
     minimap2
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#paftools">
       paftools
      </a>
     </li>
    </ul>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#winnowmap">
     Winnowmap
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#mummer">
     MUMmer
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#dnadiff">
       dnadiff
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#consensus">
   Consensus
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#racon">
     Racon
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#id1">
   Graph
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#bandage">
     Bandage
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sam-bam">
   SAM/BAM
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#samtools">
     Samtools
    </a>
   </li>
  </ul>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Tools</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#assembler">
   Assembler
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#flye">
     Flye
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#miniasm">
     miniasm
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quickmerge">
   quickmerge
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#trimmer">
   Trimmer
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#nanofilt">
     Nanofilt
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#porechop">
     Porechop
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#graph">
   Graph
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#vg">
     VG
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#minigraph">
     minigraph
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#pggb">
     pggb
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#alignment">
   Alignment
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#minimap2">
     minimap2
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#paftools">
       paftools
      </a>
     </li>
    </ul>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#winnowmap">
     Winnowmap
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#mummer">
     MUMmer
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#dnadiff">
       dnadiff
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#consensus">
   Consensus
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#racon">
     Racon
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#id1">
   Graph
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#bandage">
     Bandage
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sam-bam">
   SAM/BAM
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#samtools">
     Samtools
    </a>
   </li>
  </ul>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section id="tools">
<h1>Tools<a class="headerlink" href="#tools" title="Permalink to this headline">#</a></h1>
<p><a class="reference external" href="https://learn.gencore.bio.nyu.edu/ngs-file-formats/quality-scores/">FastQ Quality Scores</a></p>
<section id="assembler">
<h2>Assembler<a class="headerlink" href="#assembler" title="Permalink to this headline">#</a></h2>
<p>Assember…</p>
<section id="flye">
<h3>Flye<a class="headerlink" href="#flye" title="Permalink to this headline">#</a></h3>
<p>Flye is a de novo assembler.</p>
<p>bash:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="linenos">1</span>conda<span class="w"> </span>install<span class="w"> </span>-c<span class="w"> </span>bioconda<span class="w"> </span>flye
<span class="linenos">2</span>flye<span class="w"> </span>--nano-raw<span class="w"> </span><span class="se">\</span>
<span class="linenos">3</span>~/course_data/precompiled/all_guppy.fastq<span class="w"> </span><span class="se">\</span>
<span class="linenos">4</span>--genome-size<span class="w"> </span>1m<span class="w"> </span>--out-dir<span class="w"> </span>./flye_output
</pre></div>
</div>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/fenderglass/Flye">Flye github</a></p></li>
<li><p><a class="reference external" href="https://www.youtube.com/watch?v=nF83anOSYoA">Flye youtube</a></p></li>
<li><p><a class="reference external" href="https://timkahlke.github.io/LongRead_tutorials/ASS_F.html">Flye tutorial</a></p></li>
</ul>
</section>
<section id="miniasm">
<h3>miniasm<a class="headerlink" href="#miniasm" title="Permalink to this headline">#</a></h3>
<p>Miniasm is a very fast OLC-based de novo assembler for noisy long reads.</p>
<p>miniasm:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="linenos">1</span>conda<span class="w"> </span>install<span class="w"> </span>-c<span class="w"> </span>bioconda<span class="w"> </span>miniasm
</pre></div>
</div>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/lh3/miniasm">miniasm github</a></p></li>
<li><p><a class="reference external" href="https://www.youtube.com/watch?v=f4sT5pEHoxU">miniasm youtube</a></p></li>
</ul>
</section>
</section>
<section id="quickmerge">
<h2>quickmerge<a class="headerlink" href="#quickmerge" title="Permalink to this headline">#</a></h2>
<p>Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</p>
<p>quickmerge:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="linenos">1</span>conda<span class="w"> </span>install<span class="w"> </span>-c<span class="w"> </span>conda-forge<span class="w"> </span>-c<span class="w"> </span>bioconda<span class="w"> </span>quickmerge
<span class="linenos">2</span><span class="c1"># using wrapper</span>
<span class="linenos">3</span>merge_wrapper.py<span class="w"> </span>hybrid_assembly.fasta<span class="w"> </span>self_assembly.fasta
</pre></div>
</div>
</section>
<section id="trimmer">
<h2>Trimmer<a class="headerlink" href="#trimmer" title="Permalink to this headline">#</a></h2>
<section id="nanofilt">
<h3>Nanofilt<a class="headerlink" href="#nanofilt" title="Permalink to this headline">#</a></h3>
<p>Trimming and filtering Oxford Nanopore sequencing reads.</p>
<p>Nanofilt:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="linenos">1</span>NanoFilt<span class="w"> </span>–l<span class="w"> </span><span class="m">500</span><span class="w"> </span>--headcrop<span class="w"> </span><span class="m">10</span><span class="w"> </span>-q<span class="w"> </span><span class="m">10</span><span class="w"> </span>&lt;<span class="w"> </span>./Q5705/data/<span class="si">${</span><span class="nv">sample</span><span class="si">}</span>_Blockchain_v6.0.1.fastq<span class="w"> </span>&gt;<span class="w"> </span>./Q5705/trimmed/<span class="si">${</span><span class="nv">sample</span><span class="si">}</span>_Blockchain_v6.0.1.trimmed.fastq
</pre></div>
</div>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/wdecoster/nanofilt">Nanofilt github</a></p></li>
</ul>
</section>
<section id="porechop">
<h3>Porechop<a class="headerlink" href="#porechop" title="Permalink to this headline">#</a></h3>
<p>Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.</p>
<p>Porechop:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="linenos">1</span>conda<span class="w"> </span>install<span class="w"> </span>-c<span class="w"> </span>bioconda<span class="w"> </span>porechop
<span class="linenos">2</span>porechop<span class="w"> </span>-i<span class="w"> </span>input_reads.fastq.gz<span class="w"> </span>-o<span class="w"> </span>output_reads.fastq.gz
</pre></div>
</div>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/rrwick/Porechop">Porechop github</a></p></li>
</ul>
</section>
</section>
<section id="graph">
<h2>Graph<a class="headerlink" href="#graph" title="Permalink to this headline">#</a></h2>
<section id="vg">
<h3>VG<a class="headerlink" href="#vg" title="Permalink to this headline">#</a></h3>
<ul class="simple">
<li><p><a class="reference external" href="https://cpang.netlify.app/course-documentation">VG CPANG</a></p></li>
</ul>
</section>
<section id="minigraph">
<h3>minigraph<a class="headerlink" href="#minigraph" title="Permalink to this headline">#</a></h3>
<ul class="simple">
<li><p><a class="reference external" href="https://timkahlke.github.io/LongRead_tutorials/ASS_M.html">minigraph tutorial</a></p></li>
</ul>
</section>
<section id="pggb">
<h3>pggb<a class="headerlink" href="#pggb" title="Permalink to this headline">#</a></h3>
<ul class="simple">
<li><p><a class="reference external" href="https://pggb.readthedocs.io/en/latest/rst/quick_start.html">pggb github</a></p></li>
</ul>
</section>
</section>
<section id="alignment">
<h2>Alignment<a class="headerlink" href="#alignment" title="Permalink to this headline">#</a></h2>
<section id="minimap2">
<h3>minimap2<a class="headerlink" href="#minimap2" title="Permalink to this headline">#</a></h3>
<ul class="simple">
<li><p><a class="reference external" href="http://blog.genoglobe.com/2018/09/nanopore-sequencing-de-novo-assembly.html">Options [kr]</a></p></li>
<li><p><a class="reference external" href="https://korbillgates.tistory.com/197">Nanopore sequencing de novo assembly [kr]</a></p></li>
<li><p><a class="reference external" href="https://timkahlke.github.io/LongRead_tutorials/ASS_M.html">Genome Assembly with Minimap2 and Miniasm</a></p></li>
<li><p><a class="reference external" href="https://lh3.github.io/minimap2/minimap2.html">Man page</a></p></li>
</ul>
<section id="paftools">
<h4>paftools<a class="headerlink" href="#paftools" title="Permalink to this headline">#</a></h4>
<p>paftools.js is a script that processes alignments in the PAF format, such as converting between formats, evaluating mapping accuracy, lifting over BED files based on alignment, and calling variants from assembly-to-assembly alignment. This script requires the k8 Javascript shell to run. On Linux or Mac, you can download the precompiled k8 binary with:</p>
</section>
</section>
<section id="winnowmap">
<h3>Winnowmap<a class="headerlink" href="#winnowmap" title="Permalink to this headline">#</a></h3>
<p>Winnowmap is a long-read mapping algorithm optimized for mapping ONT and PacBio reads to repetitive reference sequences. Winnowmap development began on top of minimap2 codebase, and since then we have incorporated the following two ideas to improve mapping accuracy within repeats.</p>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/marbl/Winnowmap">Winnowmap github</a></p></li>
</ul>
</section>
<section id="mummer">
<h3>MUMmer<a class="headerlink" href="#mummer" title="Permalink to this headline">#</a></h3>
<p><a class="reference external" href="https://manpages.debian.org/testing/mummer/dnadiff.1.en.html#OPTIONS">Options</a></p>
<section id="dnadiff">
<h4>dnadiff<a class="headerlink" href="#dnadiff" title="Permalink to this headline">#</a></h4>
<p><a class="reference external" href="https://github.com/marbl/MUMmer3/blob/master/docs/dnadiff.README">dnadiff readme</a></p>
</section>
</section>
</section>
<section id="consensus">
<h2>Consensus<a class="headerlink" href="#consensus" title="Permalink to this headline">#</a></h2>
<section id="racon">
<h3>Racon<a class="headerlink" href="#racon" title="Permalink to this headline">#</a></h3>
<p>Racon is a standalone consensus building tool that can be coupled with a fast assembler such as miniasm, which performs de novo assembly with error prone long reads without error corrections. This dramatically cut down the time needed for sequence assembly and consensus generation. Racon stands for Rapid Consensus and it can be used for PacBio and Oxford Nanopore data.</p>
<ul class="simple">
<li><p><a class="reference external" href="https://github.com/isovic/racon">Racon github</a></p></li>
</ul>
</section>
</section>
<section id="id1">
<h2>Graph<a class="headerlink" href="#id1" title="Permalink to this headline">#</a></h2>
<section id="bandage">
<h3>Bandage<a class="headerlink" href="#bandage" title="Permalink to this headline">#</a></h3>
<p><a class="reference external" href="https://tylerbarnum.com/2018/02/26/how-to-use-assembly-graphs-with-metagenomic-datasets/">How to Use Assembly Graphs with Metagenomic Datasets</a></p>
</section>
</section>
<section id="sam-bam">
<h2>SAM/BAM<a class="headerlink" href="#sam-bam" title="Permalink to this headline">#</a></h2>
<section id="samtools">
<h3>Samtools<a class="headerlink" href="#samtools" title="Permalink to this headline">#</a></h3>
<ul class="simple">
<li><p><a class="reference external" href="https://samtools.github.io/hts-specs/SAMv1.pdf">Samtools offical doc</a></p></li>
<li><p><a class="reference external" href="https://hhj6212.github.io/biology/tech/2020/10/18/samtools.html">Samtools How to use</a></p></li>
</ul>
</section>
</section>
</section>


              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="Graph.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Graph</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="PythonBIO.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Bio with Python</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Brendan Jeon<br/>
  
      &copy; Copyright 2023, Brendan Jeon.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>